src/executables/util_cs.c

```diff
@@ -3275,51 +3280,59 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 	    {
 	      goto check_applied_info_end;
 	    }
-	  error = db_restart (arg->command_name, TRUE, local_database_name);
-	  if (error != NO_ERROR)
+	  if ((error = db_restart (arg->command_name, TRUE, local_database_name)) != NO_ERROR)
```

Comment:
db_restart() also prints error message inside. 
why do you store the error only for db_restart()?

Reply:
I cannot find any print functions inside db_restart().

Reply:
What do you mean by 'print function'? 
where is the print function in `db_login()`?
What I meant to say was that the error message was logged through er_set().

Reply:
There is no printing error message inside db_restart().

Right, no printing error message inside db_login().

like db_restart(), db_login() must store the return value. 

Is this right what you meant?

Reply:
Right. 
if you change the code, please resolve the conversations related to this.